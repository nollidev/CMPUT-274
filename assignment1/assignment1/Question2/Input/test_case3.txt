When they say 2% milk, I don't know what the other 98% is.