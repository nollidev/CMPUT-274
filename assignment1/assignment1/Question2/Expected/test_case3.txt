is 98% other the what know dont I milk 2% say they When
is 98% other the what know dont I milk 2% say they When
4
